Products/ZenUtils/Auth0/Auth0.py

```diff
@@ -118,10 +118,12 @@ def storeIdToken(id_token, session, conf):
                              audience=conf['clientid'],
                              issuer=conf['tenant'])
 
-        sessionInfo = session.setdefault(Auth0.session_key, SessionInfo())
+        sessionInfo = SessionInfo()
```

Comment:
I don't think you need to construct a new SessionInfo object every time; as long as you set the refresh_token, the old code would have worked.  If we are making changes to the SessionInfo object elsewhere that we expect to persist, then this change could break things.